lib/botRepository.js

```diff
@@ -17,17 +18,25 @@ class ChatRepository {
 
   addChat(chatId) {
     this.redis.sadd(CHATS_SET_KEY, chatId);
-  };
+  }
 
   delChat(chatId) {
     this.redis.srem(CHATS_SET_KEY, chatId);
-  };
+  }
 
   getAllChats() {
     return Rx.Observable.fromPromise(this.redis.smembers(CHATS_SET_KEY))
       .flatMap(el => el);
   }
 
+  getLastPolledDate() {
```

Comment:
I don't see the big deal in having a shared collaborator instance to be honest, but I see your point since this project isn't going to grow much more than the current status.

Up to you, not a blocker